packages/smithy/goldens/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:lints/recommended.yaml
```

Comment:
Do we want to use amplify lints? Could be done later as well.

Reply:
Nah, generated code should meet `lints` but it's not worth trying to get it cleaner than that.